drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -3876,6 +4009,10 @@ static int renesas_eth_sw_probe(struct platform_device *pdev)
 
 	priv->fib_nb.notifier_call = rswitch_fib_event;
 
+	ret = register_netdevice_notifier(&vlan_notifier_block);
```

Comment:
Nit: please, move registration of notifier before assigning rswitch_fib_event, to make code more structural.